### DISCUSSION TITLE
Dexcom ONE ("C" starting transmitter) on Wear OS collector

### DISCUSSION THREAD

**Discussion by alexb08** [NightscoutFoundation/xDrip#3034] [Bug reports] [answered]
Hello,

First, I would like to thank you for xDrip. I have been a user since 2020 and I am grateful for the flexibility it gives me.

I am trying to use collector mode on a Galaxy Watch 5 with my Dexcom ONE transmitter, but I get no reading.
I have installed the Wear OS companion app, the phone & wear apps are the same version, but it fails at the "Authorizing" step.
Since Dexcom ONE (transmitter starting with C) fix was released at the beginning of this year, on the xDrip phone app status page, I noticed that there is also an "Authorizing 2" step. But when the Wear app should be collecting, it never gets to the "Authorizing 2". 

This makes me think that the Dex1 C-transmitters plugin is maybe not available on the watch app?

Below are some logs from when the Wear collector tries to connect to the transmitter.
Big thanks for the help.

![Screenshot_20230729_192530_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/8611415/ce128567-c9db-4eb6-abf2-e619584e745c)

![Screenshot_20230730_005131_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/8611415/cd050d6c-e417-4331-9da3-b34253940da3)

**Comment by Navid200** [comment, marked as answer]
Unfortunately, there is no active xDrip developer who uses WearOS.  As a result, the WearOS tickets take a backseat.

Everyone is welcome to open PRs.  So, if any developer likes to fix this, or any other WearOS shortcoming, they are more than welcome to open submissions (PRs).
